BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -663,41 +653,108 @@ extension AttributedLabel {
     struct Link: Equatable, Hashable {
         var url: URL
         var range: NSRange
+        weak var container: LabelView?
+
+        var boundingRect: CGRect {
+            guard let container = container,
+                  let textStorage = container.makeTextStorage(),
+                  let layoutManager = textStorage.layoutManagers.first,
+                  let textContainer = layoutManager.textContainers.first
+            else {
+                return .zero
+            }
+
+            let glyphRange = layoutManager.glyphRange(forCharacterRange: range, actualCharacterRange: nil)
+
+            // In cases where a link overflows from one line to the next,
+            // we will get back more than one rect. If we had used the boundingRect
+            // function instead, it will only return us a bounding rect that can
+            // fully encompass all the sub-rects - this is undesired behavior as
+            // it will suppress focusable items within the label that have an
+            // overlap with the larger bounding box.
+            // Since we only select the first rectangle, if there is indeed an overflow,
+            // only the portion of the link in the preceding line would be highlighted by
+            // VoiceOver or Full Keyboard Access. While this is not ideal behavior, it is still
+            // preferred over the former. The correct fix for this would be to return bezier paths
+            // that encompass exactly the linked portion.
+            let rects = { () -> [CGRect] in
+                var glyphRects: [CGRect] = []
+                layoutManager.enumerateEnclosingRects(
+                    forGlyphRange: glyphRange,
+                    withinSelectedGlyphRange: NSMakeRange(NSNotFound, 0),
+                    in: textContainer
+                ) { rect, _ in
+                    glyphRects.append(rect)
+                }
+                return glyphRects
+            }()
+
+            return rects.first ?? .zero
```

Comment:
Not sure if this will compile but I think using the second block parameter would make this code simpler
```suggestion
            var rect: CGRect?
            layoutManager.enumerateEnclosingRects(
                forGlyphRange: glyphRange,
                withinSelectedGlyphRange: NSMakeRange(NSNotFound, 0),
                in: textContainer
                ) { rect, stop in
                    glyphRects.append(rect)
                    stop = true
            }
            return rect ?? .zero
```